### PR TITLE
Update examples

### PR DESCRIPTION
**Pull request by starek4**
Examples are an important part of every open source project. We should have them up to date.

### CODE REVIEW

**Comment by starek4** [comment]
`stickler-ci` looks confused, ignore him 😃

File: examples/ftp_simple.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
-#!/usr/bin/env python
-# Designed for use with boofuzz v0.0.8
+#!/usr/bin/env python3
+# Designed for use with boofuzz v0.1.7
```
Comment (SR4ven) [inline examples/ftp_simple.py]: Next release will be 0.2.0 as we introduce the new connection classes with backwards compatibility.

File: examples/http_simple.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+    s_initialize(name="Request")
+    with s_block("Request-Line"):
+        s_group("Method", ["GET", "HEAD", "POST", "PUT", "DELETE", "CONNECT", "OPTIONS", "TRACE"])
+        s_delim(" ", name="space-1")
+        s_string("/index.html", name="Request-URI")
+        s_delim(" ", name="space-2")
+        s_string("HTTP/1.1", name="HTTP-Version")
+        s_static("\r\n", name="Request-Line-CRLF")
+    s_static("\r\n", "Request-CRLF")
```
Comment (SR4ven) [inline examples/http_simple.py]: Host-Line is missing.
Currently this will result in a 400 Bad Request.

**Comment by starek4** [comment]
Yes, I have no problem with that 👍

**Comment by SR4ven** [comment]
Thank you @starek4!

**Pull request merged**